sample_construct_graphs/one_gateway_two_lambda.yaml

```diff
@@ -0,0 +1,22 @@
+constraints:
```

Comment:
instead could you start a test folder where we can have inputs outputs? that ways we have samples we can run through the engine with expected outputs